laser_runner_removal/laser_runner_removal/laser.py

```diff
@@ -14,44 +14,47 @@
 import numpy as np
 
 import ctypes
-import time 
-import logging 
+import time
+import logging
 
-#Define point structure for HeliosDac
+
+# Define point structure for HeliosDac
 class HeliosPoint(ctypes.Structure):
-    _fields_ = [('x', ctypes.c_uint16),
-                ('y', ctypes.c_uint16),
-                ('r', ctypes.c_uint8),
-                ('g', ctypes.c_uint8),
-                ('b', ctypes.c_uint8),
-                ('i', ctypes.c_uint8)]
+    _fields_ = [

```

Comment:
Are you using a particular auto-formatter? I have no strong opinions, so if so I can get the same one so whitespace changes don't become common.

Reply:
Ah forgot to mention, I'm using the Black formatter via VS code, with default settings